runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityPlugin.java

```diff
@@ -185,6 +195,12 @@ public void onExperienceChanged(ExperienceChanged event)
 		int skillGained = agilityXp - lastAgilityXp;
 		lastAgilityXp = agilityXp;
 
+		int newAgilityLevel = Experience.getLevelForXp(agilityXp);
```

Comment:
This entire block is just `agilityLevel  = Experience.getLevelForXp(agilityXp);`